packages/manager/modules/pci/src/projects/project/storages/databases/add/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,25 @@
+{
+  "pci_database_add_title": "Créer une base de données",
+  "pci_database_add_description": "Créez une base de données disponible dans votre projet Public Cloud. Si vous souhaitez utiliser un réseau privé et que celui-ci n’existe pas encore, veuillez <a href='{{createPrivateNetworkLink}}'>créer un réseau privé</a> ou un sous-réseau d’abord.",
+  "pci_database_add_step1_title": "Choisissez votre base de données",
```

Comment:
I noticed sometimes we use (ez) and sometimes (er).
I think we must be consistent.

Reply:
Everything has been changed to 'er'
Thanks for your help!